packer/instance-scripts/init-quorum.sh

```diff
@@ -112,9 +116,7 @@ user=ubuntu" | sudo tee /etc/supervisor/conf.d/cloudwatch-metrics-supervisor.con
 function generate_cloudwatch_block_metrics_supervisor_config {
     local RPC_DNS=$1
     local RPC_PORT=$2
-
     local NETWORK_ID=$(cat /opt/quorum/info/network-id.txt)
```

Comment:
Leave the whitespace lines around this please

Reply:
New commit address this & one below!